wrappers/kn/src/nativeMain/kotlin/zxingcpp/ImageView.kt

```diff
@@ -0,0 +1,72 @@
+package zxingcpp
+
+import cnames.structs.zxing_ImageView
+import kotlinx.cinterop.CPointer
+import kotlinx.cinterop.ExperimentalForeignApi
+import kotlinx.cinterop.addressOf
+import kotlinx.cinterop.pin
+import zxingcpp.cinterop.*
+
+@OptIn(ExperimentalForeignApi::class)
+abstract class ImageView {
+	abstract val data: UByteArray
+	open val dataOffset: Int = 0
+	abstract val left: Int
```

Comment:
This cropping feature from the Android wrapper is also too specific to the android use case. Please only model the members of the c-API version.

Reply:
fixed in https://github.com/zxing-cpp/zxing-cpp/pull/719/commits/c4cc120fb9f1948685e7bbfe1f1e0c7c163e20b1